core.py

```diff
@@ -31,20 +30,37 @@
 from scripts.list_dataset_metadata_handler import list_dataset_metadata_handler
 from version import __version__
 
+VALIDATION_SUPPORTED_FORMATS = ["XPT", "JSON", "NDJSON"]
```

Comment:
Instead of creating a new list of formats you should use the existing DataFormatTypes. You could add the file extension information to the DataFormatTypes values.